fcli-core/fcli-sc-sast/src/main/java/com/fortify/cli/sc_sast/_common/session/cli/mixin/SCSastSessionLoginOptions.java

```diff
@@ -21,7 +21,7 @@
 
 public class SCSastSessionLoginOptions {
     @ArgGroup(exclusive = false, multiplicity = "1", order = 1)
-    @Getter private SCSastUrlConfigOptions urlConfigOptions = new SCSastUrlConfigOptions();
+    @Getter private SCSastAndSSCUrlConfigOptions urlConfigOptions = new SCSastAndSSCUrlConfigOptions();
```

Comment:
For consistency/clarity, please change variable name to something like `scSastAndSscUrlConfigOptions`